sunpy/coordinates/transformations.py

```diff
@@ -361,4 +356,18 @@ def hgs_to_hgs(from_coo, to_frame):
         return from_coo.transform_to(HCRS).transform_to(to_frame)
 
 
+@frame_transform_graph.transform(FunctionTransform, Heliocentric, Heliocentric)
+def hcc_to_hcc(from_coo, to_frame):
```

Comment:
To be consistent with the other transform functions, I think the input args should be `hcccoord` and `hccframe`. I realize that the `hgs_to_hgs` transform uses this same naming, though I would claim that that should be changed as well.